### ISSUE TITLE
Change icons to use spans instead of i tags

### ISSUE THREAD

**Issue by Rambomst** [fraywing/textAngular#1166]
In order for this editor to be WCAG 2.0 compliant you can't use `<i>` tags for icons since it doesn't follow the WCAG 2.0 guidelines. Instead you should be using `<span>` tags.

**Comment by JoelParke** [comment]
Perhaps you can submit a PR with the changes you suggest.
